spec/models/order_spec.rb

```diff
@@ -10,4 +10,36 @@
     it { should have_many :order_items }
     it { should have_many(:items).through(:order_items) }
   end
+
+  describe 'Instance Method(s)' do
+    it '#total_quantity' do
+      oi_1 = create(:order_item)
+      #creates three order_items and items bound to same order from oi_1
+      oi_2, oi_3, oi_4 = create_list(:order_item, 3, order: oi_1.order)
+      #main order for clarity in test
+      order_1 = oi_1.order
+
+      #total quantity of items in a single order
+      actual = order_1.total_quantity
+      #generated via pry session AR query
+      expected = 28
```

Comment:
We should not hard code this.  It's probably why the test is passing even though .total_quantity is calling stock of items